### PR TITLE
Add sights to trip

### PR DESCRIPTION
**Pull request by VictoriaWika**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,18 @@
...
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <title>Travelr</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (macfrei) [inline public/index.html:30]: I think I'd keep that in.

File: public/index.html [hunk 2]
```diff
@@ -2,42 +2,18 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="./location.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
```
Comment (macfrei) [inline public/index.html:10]: You could change the description

File: src/components/CityPage/CityPage.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import Button from '../Button/Button'
+import GoBackButton from '../GoBackButton/GoBackButton'
+import { ReactComponent as PlusSVG } from '../../icons/plus.svg'
```
Comment (macfrei) [inline src/components/CityPage/CityPage.js:5]: Yay! Love this!

**Review by macfrei — COMMENTED** [review]
Very nice! 🚀

File: public/index.html (hunk 2 above)
Comment (VictoriaWika) [reply public/index.html:10]: Done! 👍🏻

**Pull request merged**